staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -602,15 +602,21 @@ func (q *Quantity) Neg() {
 // of most Quantity values.
 const int64QuantityExpectedBytes = 18
 
-// String formats the Quantity as a string, caching the result if not calculated.
+// String formats the Quantity as a string. The unbuffered version exists to provide human readable
+// log format for structs/maps that contain nested Quantity field.
+func (q Quantity) String() string {
+	result := make([]byte, 0, int64QuantityExpectedBytes)
+	number, suffix := q.CanonicalizeBytes(result)
+	number = append(number, suffix...)
+	return string(number)
+}
+
+// StringBuffered formats the Quantity as a string, caching the result if not calculated.
```

Comment:
Why `*Buffered`?  We don't really have a convention for this sort of operation today via naming, but I might lean towards `FastString()` or `AsString()`.

Will come back to this after code freeze.